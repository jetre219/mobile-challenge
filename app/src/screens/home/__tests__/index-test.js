import { render } from '@testing-library/react-native';
import React from 'react';
import HomeScreen from '..';

let homeScreen;

describe('Home screen', () => {
  beforeEach(() => {
    homeScreen = render(<HomeScreen />);
  });

  it('renders correctly', () => {
    expect(homeScreen.toJSON()).toMatchSnapshot();
  });
});
