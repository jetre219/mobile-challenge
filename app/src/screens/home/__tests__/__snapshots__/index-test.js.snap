// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#242424",
        "fontSize": 27.5,
      }
    }
  >
    Home Screen
  </Text>
</View>
`;
